### PR TITLE
fix: code review comments on mobile

### PR DESCRIPTION
**Pull request by kolaente**
This PR fixes code review comments on mobile.

#### Before:
Screen Shot 2021-10-15 at 18 21 14

#### After:
Screen Shot 2021-10-15 at 19 06 38

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

File: templates/repo/issue/view_content/comments.tmpl [hunk 1]
```diff
@@ -599,7 +599,7 @@
 												</div>
 											{{end}}
 										</div>
-										<div>
+										<div class="code-comment-buttons-buttons">
```
Comment (delvh) [inline templates/repo/issue/view_content/comments.tmpl:602]:
```suggestion
										<div class="code-comment-buttons">
```

Comment (kolaente) [reply templates/repo/issue/view_content/comments.tmpl:602]: `code-comment-buttons` already exists and is applied on the parent div. I needed some class to use here that wasn't `buttons` or something else that's already used.

Comment (delvh) [reply templates/repo/issue/view_content/comments.tmpl:602]: So this is intended?
I thought that was simply an unwanted duplication…

Comment (kolaente) [reply templates/repo/issue/view_content/comments.tmpl:602]: Yes, it is intended. Open for suggestions though, feels like a hack :slightly_smiling_face:

**Review by delvh — APPROVED** [review]

File: templates/repo/issue/view_content/comments.tmpl (hunk 1 above)
Comment (delvh) [reply templates/repo/issue/view_content/comments.tmpl:602]: Can't think of a better solution right now, so I'll approve it. But I know what you mean, it certainly does not feel like a "correct" solution.

**Pull request merged**